benchmarks/Cargo.toml

```diff
@@ -0,0 +1,29 @@
+[package]
+name = "benchmarks"
+version = "0.1.0"
+edition.workspace = true
+rust-version.workspace = true
+license.workspace = true
+authors.workspace = true
+homepage.workspace = true
+repository.workspace = true
+exclude.workspace = true
+
+[[bin]]
+name = "benchmarks"
+path = "src/main.rs"
+required-features = ["executable"]
+
+[features]
+default = ["std"]
+std = ["miden-lib/std", "miden-objects/std", "miden-tx/std", "vm-processor/std", "mock/std", "serde_json/std"]
+executable = ["std"]
```

Comment:
This crate can be just an executable and so:

- I don't think we need `required-features` here.
- We probably also don't need to support `no_std` - so, the whole `features` section can probably go away.

Reply:
For some reason, if I try to change anything in this setup, the `no-std` build starts to fail with the same `the wasm*-unknown-unknown targets are not supported by default, you may need to enable the "js"` error. I still don't understand why it works like that, let me investigate once more.